spec/factories/legal_aid_applications.rb

```diff
@@ -9,7 +9,7 @@
       transient do
         with_bank_accounts { 0 }
       end
-      applicant { create :applicant, with_bank_accounts: with_bank_accounts }
+      applicant { create :applicant, with_bank_accounts: with_bank_accounts, date_of_birth: Date.new(2000, 1, 1) }
```

Comment:
I don't think you should fix this date_of_birth here.
See my next comment for an alternative